transport/x/grpc/handler.go

```diff
@@ -59,9 +59,87 @@ func (h *handler) handle(srv interface{}, serverStream grpc.ServerStream) error
 		return errInvalidGRPCStream
 	}
 
+	streamMethod := stream.Method()
+	transportRequest, err := h.getBasicTransportRequest(ctx, streamMethod)
+	if err != nil {
+		return err
+	}
+
+	handlerSpec, err := h.i.router.Choose(ctx, transportRequest)
+	if err != nil {
+		return err
+	}
+	switch handlerSpec.Type() {
+	case transport.Unary:
+		return h.handleUnary(ctx, transportRequest, serverStream, streamMethod, start, handlerSpec.Unary())
+	case transport.Stream:
+		return h.handleStream(ctx, transportRequest, serverStream, handlerSpec.Stream())
+	}
+	return yarpcerrors.UnimplementedErrorf("transport grpc does not handle %s handlers", handlerSpec.Type().String())
```

Comment:
This needs to be converted back

Reply:
There's a bigger overall issue here - streaming isn't handling yarpcerrors. In the code before this PR, everything was done within `handleBeforeErrorConversion`, and a response metadata was used to handle named errors. None of this logic applies to streaming now, so this needs to be weaved into streaming error handling.

Reply:
We also need to handle opentracing before this can be merged.